guidance_note_wb.md

```diff
@@ -7,102 +7,149 @@ title: "Guidance to Authors"
-This includes manually constructed or derivative datasets drawn from public sources such as the World Development Indicators, where the primary purpose of the dataset is to facilitate replication of the paper.
-If there are no restrictions on publishing the derivative dataset(s), data files may be published directly in the reproducibility package.
-This also includes dummy datasets produced when access restrictions do not allow for publication of the data itself, but dummy datasets are useful to verify the functionality of the published code.
-
-The reproducibility package will be published concurrently with the publication.
-The cover page of the publication will include the Reproducible Research Seal to indicate that a verified reproducibility package is available, and provide the permanent URL for the package.
-The publication, the reproducibility package in the Reproducible Research Repository, and any data sources in the MicroData Library or Development Data Hub will all be cross-linked to maximize discoverability.
+Once a reproducibility package is verified, the reviewer prepares it for publication in the Reproducible Research Repository at [reproducibility.worldbank.org](https://reproducibility.worldbank.org).
+This includes preparing the metadata and drafting the catalogue entry.
+
+Each published reproducibility package will include:  
+
+- A README file with instructions for running the package and a clear Data Availability Statement (DAS)  
+- A zipped folder containing all code files and, where permissible, the data files  
+- A license file  
+- The reproducibility report with details of the verification  
+- A file with the SHA256 hashes of all datasets in the data folder, to enable integrity checks  
+- A comparison report documenting any differences between the datasets used by the authors and the datasets currently available (applicable for Limited Access or Accessible data types)  
+
+Every reproducibility package will be assigned a DOI to support discovery, cross-linking, and citation.  
+
+### Treatment of Data in Published Packages  
+Depending on access restrictions, data in the reproducibility package will take one of the following forms:  
+
+1. **Data already published (internally or externally):**  
+   The Data Availability Statement (DAS) links to the data with access instructions and version details. 
+   If permitted, the data may also be included directly in the package, which is recommended when no DOI or stable link exists. ch is recommended when no DOI or stable link exists.  
+
+2. **Proprietary or confidential data:**  
+   The DAS explains how third parties may access the data. 
+   Input data will not be included, though derivative datasets may be shared if allowed. 
+   Data available to Bank staff should also be deposited in the internal catalogue and linked in the DAS.  
+
+3. **World Bank–owned unpublished data:**  
```

Comment:
This isn't very clear. I'm not sure how it's different from case 1. maybe this is the case of forthcoming or embargoed data, where we still publish the package?